ui/components/app/network-connection-banner/network-connection-banner.tsx

```diff
@@ -29,23 +32,106 @@ type BannerIcon = {
   className?: string;
 };
 
+const renderPrimaryMessage = ({
```

Comment:
Nit: These components are essentially React components (takes props, returns a component). Should they be used as such?


```suggestion
const PrimaryMessage = ({
```